packages/preview/qooklet/0.1.0/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 ivaquero
```

Comment:
The year here is outdated I think (but this is not that important).

Reply:
This is the start date of the repo, I'll update it